app/src/main/java/com/capstone/weatherapp/WeatherCache.kt

```diff
@@ -14,25 +14,34 @@ data class CityCache(
     val temp_max: Double,
     val humidity: Double
 ) {
-    fun convertToWeatherApiResponse(cityCacheList: List<CityCache>) : WeatherApiResponse {
-        val cityList = mutableListOf<City>()
-        for (cityCache: CityCache in cityCacheList) {
-            val city = City(
-                id = cityCache.id,
-                name = cityCache.name,
-                coord = Coord(constants.DOUBLE_ZERO, constants.DOUBLE_ZERO),
-                weather = emptyList(),
-                main = Main(temp=temp, feels_like=temp, temp_min=temp_min, temp_max=temp_max, pressure=constants.DOUBLE_ZERO, humidity=humidity),
-                visibility = constants.INT_ZERO,
-                wind = Wind(constants.DOUBLE_ZERO, constants.INT_ZERO),
-                clouds = Clouds(constants.INT_ZERO),
-                dt = constants.INT_ZERO,
-                sys = Sys(country=country, timezone=0, sunrise=0, sunset=0)
-            )
-            cityList.add(city)
-        }
+    companion object {
```

Comment:
Why did you move this inside a companion object?

Reply:
seemed like more of a static class level function to me; we're passing in a list of CityCache objects to be converted, so it's not exactly object specific

Reply:
Gotcha. If you take it out of the companion object you should just be able to call the function on any instances of the data class. For example:
```
val weatherCache: WeatherCache
weatherCache.convertToWeatherApiResponse()
```
Then you don't need to pass in any parameters to the method and it's a lot cleaner call. You don't need to fix it to this, just wanted to point it out